sample-apps/flask_sample/bin/test

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+poetry run python -m unittest
```

Comment:
I think in the future we probably don't need tests for the sample apps. I know there were already tests in place here, so this is good for this PR.